storageapi/src/vespa/storageapi/messageapi/messagehandler.h

```diff
@@ -120,215 +119,81 @@ class RemoveLocationReply;
-            const std::shared_ptr<api::SetSystemStateCommand>&)
-        { return false; }
-    virtual bool onSetSystemStateReply(
-            const std::shared_ptr<api::SetSystemStateReply>&)
-        { return false; }
-    virtual bool onActivateClusterStateVersion(
-            const std::shared_ptr<api::ActivateClusterStateVersionCommand>&)
-    { return false; }
-    virtual bool onActivateClusterStateVersionReply(
-            const std::shared_ptr<api::ActivateClusterStateVersionReply>&)
-    { return false; }
-    virtual bool onGetSystemState(
-            const std::shared_ptr<api::GetSystemStateCommand>&)
-        { return false; }
-    virtual bool onGetSystemStateReply(
-            const std::shared_ptr<api::GetSystemStateReply>&)
-        { return false; }
-    virtual bool onBucketsAdded(
-            const std::shared_ptr<api::BucketsAddedCommand>&)
-        { return false; }
-    virtual bool onBucketsAddedReply(
-            const std::shared_ptr<api::BucketsAddedReply>&)
-        { return false; }
-    virtual bool onBucketsRemoved(
-            const std::shared_ptr<api::BucketsRemovedCommand>&)
-        { return false; }
-    virtual bool onBucketsRemovedReply(
-            const std::shared_ptr<api::BucketsRemovedReply>&)
-        { return false; }
-
-    virtual bool onStatBucket(
-            const std::shared_ptr<api::StatBucketCommand>&)
-        { return false; }
-    virtual bool onStatBucketReply(
-            const std::shared_ptr<api::StatBucketReply>&)
-        { return false; }
-
-    virtual bool onGetBucketList(
-            const std::shared_ptr<api::GetBucketListCommand>&)
-        { return false; }
-    virtual bool onGetBucketListReply(
-            const std::shared_ptr<api::GetBucketListReply>&)
-        { return false; }
-
-    virtual bool onRemoveLocation(
-            const std::shared_ptr<api::RemoveLocationCommand>&)
-        { return false; }
-
-    virtual bool onRemoveLocationReply(
-            const std::shared_ptr<api::RemoveLocationReply>&)
-        { return false; }
+    // Basic operations
+    virtual bool onGet(const std::shared_ptr<api::GetCommand>&) { return false; }
+    virtual bool onGetReply(const std::shared_ptr<api::GetReply>&) { return false; }
+    virtual bool onPut(const std::shared_ptr<api::PutCommand>&) { return false; }
+    virtual bool onPutReply(const std::shared_ptr<api::PutReply>&) { return false; }
+    virtual bool onUpdate(const std::shared_ptr<api::UpdateCommand>&) { return false; }
+    virtual bool onUpdateReply(const std::shared_ptr<api::UpdateReply>&) { return false; }
+    virtual bool onRemove(const std::shared_ptr<api::RemoveCommand>&) { return false; }
+    virtual bool onRemoveReply(const std::shared_ptr<api::RemoveReply>&) { return false; }
+    virtual bool onRevert(const std::shared_ptr<api::RevertCommand>&) { return false; }
+    virtual bool onRevertReply(const std::shared_ptr<api::RevertReply>&) { return false; }
+
+    // Visiting
```

Comment:
Consider removing this comment, as it does not apply to the whole block of declarations below it

Reply:
Fixed